src/components/EditObject.vue

```diff
@@ -84,11 +83,10 @@ export default {
           this.isActive.value = false;
         })
         .catch(error => {
-          console.log(error);
           this.errors = error.response.data.detail
-            .filter(detail => detail.loc[1] !== "type")
+            .filter(detail => detail["loc"][2].toLowerCase() === this.typeDefinition.type)
             .map(detail => {
-              return { field: detail.loc[1], message: detail.msg };
+              return { field: detail["loc"][3], message: detail.msg };
```

Comment:
just for my own understanding. Why are you now using 2 for type checking and returning 3 for field?

Reply:
FastAPI returns a weird structure that I had to guess how to parse. the [2] item is which Class the error is comming from (Regexp, ForensicArtifact) and the [3] item is which field in the Class is giving that error.